airbnb_project/listings/harvester_app/harvester/custom_settings.py

```diff
@@ -1,25 +1,30 @@
 import os
 
+from dotenv import dotenv_values
```

Comment:
`dotenv_values` not used after this import

Reply:
removed unused import